Lesson30/Firat/my-app/src/components/Exercise2/index.jsx

```diff
@@ -0,0 +1,15 @@
+import { useState } from "react";
+
+export const Exercise2 = () => {
+  const [label, setLabel] = useState(false);
```

Comment:
Since you need to toggle between label "On" and "Off", the most reasonable is to start with the value "On" for the label `  const [label, setLabel] = useState("On");`. You don't need to introduce a new type as it's right now (false for the initial value).